### PR TITLE
feat: agent install name

### PR DESCRIPTION
**Pull request by robot9706**
Implemented the option to change the container name of dagent, to avoid name collision when installing multiple agents on the same node.

### CODE REVIEW

**Comment by nandor-magyar** [comment]
The same collision is true for crane as well. The name `workload` would be more appropriate to address the running instances of both environments.

File: web/crux-ui/src/models/node.ts [hunk 1]
```diff
@@ -75,6 +75,7 @@ export type NodeGenerateScript = {
   rootPath?: string
   scriptType: NodeInstallScriptType
   dagentTraefik?: NodeInstallTraefik
+  workloadName?: string
```
Comment (m8vago) [inline web/crux-ui/src/models/node.ts:78]: Please add it to the validation schema as well.

File: web/crux/assets/install-script/install-docker.ps1.hbr [hunk 2]
```diff
@@ -53,7 +53,7 @@ docker run `
   {{/if}}
   -e HOST_DOCKER_SOCK_PATH=//var/run/docker.sock `
   --add-host=host.docker.internal:host-gateway `
-  --name 'dagent' `
+  --name '{{#if workloadName}}{{workloadName}}{{else}}dagent{{/if}}' `
```
Comment (m8vago) [inline web/crux/assets/install-script/install-docker.ps1.hbr:56]: would be nicer imho, if we could refrain to use handlebar conditions and move this if to the ts side. so the template simply can be workloadName

**Review by polaroi8d — CHANGES_REQUESTED** [review]
Please implement a container name validation. 

After naming the agent to: `teszting 0001 ?#@$_+)!@31 `, got the following command after running the curl command: 
```
docker: Error response from daemon: Invalid container name (teszting 0001 ?#@$_+)!@31 ), only [a-zA-Z0-9][a-zA-Z0-9_.-] are allowed.
```

**Review by m8vago — APPROVED** [review]

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**